.travis.yml

```diff
@@ -15,16 +15,18 @@ php:
   - hhvm-nightly
 
 matrix:
-    allow_failures:
-        - php: nightly
-        - php: hhvm-nightly
-    fast_finish: true
+  include:
+    - php: 5.4
+      env: COMPOSER_FLAGS="--prefer-lowest"
+  allow_failures:
+    - php: nightly
+    - php: hhvm-nightly
+  fast_finish: true
 
 before_script:
-    - composer self-update
-
-install:
-  - composer install --dev
+  - composer self-update
+  - composer config -g github-oauth.github.com $GITHUB_OAUTH_TOKEN
```

Comment:
If I remove it, dist-package and cache will not work.

Please see also https://github.com/dunglas/php-property-info/pull/2#issuecomment-125619271.